Pipfile

```diff
@@ -43,7 +43,7 @@ kombu = "<5.3"  # https://issues.redhat.com/browse/COST-3997
 msrestazure = "*"
 numpy = {version = "*", markers = "platform_machine == 'aarch64' or platform_machine == 'arm64' or platform_machine == 'x86_64'"}
 oci = "*"
-pandas = "*"
+pandas = "<2.2"
```

Comment:
Can we add a comment about why this constraint is needed so we'll know when it's ok to remove it?